review_checklist.rst

```diff
@@ -0,0 +1,52 @@
+****************
+Review Checklist
+****************
+
+When reviewing, keep in mind that we follow `Ansible Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_ in all our contributions and interactions within this repository.
```

Comment:
```suggestion
When reviewing, keep in mind that we follow the `Ansible Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_ in all our contributions and interactions within this repository.
```

Reply:
For the checklist, each should start with a capital letter and end in a period (like a sentence).

Reply:
ok, thanks, will fix